macros/misc/randomNamesPronouns.pl

```diff
@@ -0,0 +1,729 @@
+
+=head1 NAME
+
+randomNamesPronouns.pl - Load macros for random names.
+
+=head2 SYNOPSIS
+
+    loadMacros('randomNamesPronouns.pl');
+
+=head2 DESCRIPTION
+
+C<randomNamesPronouns.pl> provides a randomName function that generates a random
+name with subject, possessive, adjectives and object pronouns
+with and without capitilization and verb conjugation.
+
+Note: This idea and the names were taken from the C<PCCmacros.pl> C<RandomName>
+subroutine to extend to the handling of pronouns. The first names were generated by randomly
```

Comment:
```suggestion
subroutine to extend to the handling of pronouns.
```

Reply:
i repeatedly found myself reading "first names" to mean the first several names on the list below. Here are some edits to make it more clear what is meant.